docs/embedding/static-embedding-parameters.md

```diff
@@ -115,6 +118,8 @@ You can use locked parameters to display filtered data based on attributes captu
 
 Locked parameters will apply the selected filter values to your original dashboard or SQL question, but they won't be displayed as filter widgets on your embed. Locked parameters may also limit the values that are shown in your [editable filter widgets](#adding-a-filter-widget-to-a-static-embed).
 
+If you just want to require a value for the parameter, see [Always require a value](../dashboards/filters.md#requiring-a-filter).
```

Comment:
Maybe we should mention that in this case they most likely want to make the filter editable and that it also works with Locked params as well?